test/e2e/kubernetes/scripts/auditd-validate.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -x
+systemctl status auditd
```

Comment:
we could possibly generalize this (make `auditd` an arg) and reuse it for other services

Reply:
True, I wanted to isolate it for now so it's easier to wrap inside an "ubuntu distro only" condition. I think we should probably do something like this in `ubuntu-installed-packages-validate.sh` once auditd is in the VHD.